compiler/daml-lf-verify/README.md

```diff
@@ -0,0 +1,248 @@
+# DAML Verification Tool
```

Comment:
Note that our public-facing docs published on docs.daml.com are written in Rst in the `docs` directory. We can keep it in markdown for this PR but once we ship it in the SDK we need to change that.

Reply:
Ah darn it, I forgot about that. I'll quickly update it :)

Reply:
Converting it to rst would be great but I would leave it out of the docs directory until it is bundled in the SDK (which I would leave out of this PR).